src/components/EventOverview/EventsPage.tsx

```diff
@@ -42,6 +42,7 @@ const EventsPage = () => {
   const classes = useStyles();
   const location = useLocation<LocationState>();
   const { addedEventId } = location.state || { addedEventId: null };
+  const [archivedEventId, setArchivedEventId] = useState('');
```

Comment:
nit? - is there a reason we use empty string instead of `null` here?

Reply:
typescript doesn't let you use null, because I set it to whatever the new archived event's id on this page which a type string